modules/ocf_kubernetes/manifests/package/first_stage.pp

```diff
@@ -1,4 +1,4 @@
-class ocf::packages::kubernetes::apt {
+class ocf_kubernetes::package::first_stage {
```

Comment:
I'd switch back, `apt` indicates better than it has to do with installing the package than `first_stage`

Reply:
Yeah, I'd agree on this